doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -247,7 +247,7 @@ architecture and threat model for your application.
 
    - The previously highlighted technologies rely on the kernel, as a secure component, to enforce such policies. Because of this, we strongly recommend enabling secure boot for the Service VM, and extend the secureboot chain to any post-launched VM kernels.
    - To ensure no malicious software is introduced or persists, utilize the filesystem (FS) verification methods on every boot to extend the secure boot chain for post-launch VMs (kernel/FS).
-   - Reference: ACRN secure boot extension guide (`ClearLinux <https://projectacrn.github.io/latest/tutorials/enable_laag_secure_boot.html?highlight=secure%20boot>`_, `Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
+   - Reference: ACRN secure boot extension guide (`Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
```

Comment:
For references to ACRN documents (on projectacrn.github.io) we should use ``:ref:`doc_title` `` instead of hard coding links to ``https://projectacrn.github.io/lastest/...``.  If you use the ``https://...`` link that would link to the wrong document when we make the release-specific documentation version (e.g., for the 2.1 release).  For references to documentation to other sites, such as on the  kernel.org site, we don't have a choice and need to link to their documentation using ``https://..``

So, line 250 should be:
```
   - Reference: :ref:`how-to-enable-secure-boot-for-windows`
```

Reply:
Thank you for the explanation.